docs/src/Button.doc.js

```diff
@@ -232,6 +232,32 @@ card(
   />,
 );
 
+card(
+  <MainSection name="Usage guidelines">
```

Comment:
@ayeshakmaz The one nit I have that should in no way block this shipping is to add the name of the component in the title so we can rely on that as a title in our Gestalt docs Figma plugin.

Reply:
Chatted on Slack, but we're going to wait for this change for now